pallets/asset/src/lib.rs

```diff
@@ -426,6 +436,13 @@ decl_storage! {
                 };
                 ClassicTickers::insert(&import.ticker, classic_ticker);
             }
+
+            let sys_did = &config.reserved_country_currency_codes[0].0;
```

Comment:
Also please add a comment here to clearly separate the classic ticker logic from the currency logic, or move them into different functions.